nodeup/pkg/model/crictl.go

```diff
@@ -32,19 +33,22 @@ type CrictlBuilder struct {
 var _ fi.NodeupModelBuilder = &CrictlBuilder{}
 
 func (b *CrictlBuilder) Build(c *fi.NodeupModelBuilderContext) error {
-	assetName := "crictl"
-	assetPath := ""
-	asset, err := b.Assets.Find(assetName, assetPath)
-	if err != nil {
-		return fmt.Errorf("unable to locate asset %q: %w", assetName, err)
+	assets := b.Assets.FindMatches(regexp.MustCompile(`^crictl$`))
+	if len(assets) == 0 {
+		return fmt.Errorf("unable to find any crictl binaries in assets")
```

Comment:
Having `crictl` (and maybe `nerdctl` also) installed is not exactly a blocker.
Could this be changed to warning and return early instead? Same thing below.

Reply:
I changed: https://github.com/kubernetes/kops/pull/16486/commits/ec47b3fea6b76cf6973a21e546f9d4c1db6cb864

Reply:
Thanks! 😄